### PR TITLE
added embedding extraction use case

### PR DESCRIPTION
**Pull request by rsnm2**
@bfineran - docs site content for embedding extraction; draft

Let me know when you have some time tomorrow to fill in the details

### CODE REVIEW

**Review by dbarbuzzi — DISMISSED** [review]

File: src/content/use-cases/embedding-extraction.mdx [hunk 1]
```diff
@@ -0,0 +1,181 @@
+---
+title: "Embedding Extraction"
+metaTitle: "Embedding Extraction Deployment"
+metaDescription: "Generalized Embedding Extraction Deployment"
+index: 5000
+---
+
+# Deploying Embedding Extraction Models with DeepSparse
+
+This page explains how to deploy an Embedding Extraction pipeline with DeepSparse.
```
Comment (jeanniefinks) [inline src/content/use-cases/embedding-extraction.mdx:10]:
```suggestion
This page explains how to deploy an Embedding Extraction Pipeline with DeepSparse.
```

**Review by jeanniefinks — DISMISSED** [review]
1 p to P edit and I will approve this as a minor change

**Review by jeanniefinks — DISMISSED** [review]

**Review by bfineran — CHANGES_REQUESTED** [review]
our repos standardize to not include documentation or example files directly under our `src/` directory.

Let's move these to `examples/embedding-extraction` OR potentially `src/deepsparse/pipelines/docs/embedding-extraction`

**Review by jeanniefinks — APPROVED** [review]
Reads well, no further comments here @rsnm2

**Review by bfineran — APPROVED** [review]

**Pull request merged**